docs/tutorials/services/data-mlmodel-tutorial.md

```diff
@@ -0,0 +1,322 @@
+
+### Tag images
+
+1. From the [**DATA** page](https://app.viam.com/data/view) in the Viam app, select an image captured from your smart machine that you would like to tag.
+1. In the **Tags** field on the right-hand side, enter a new tag describing the object in the image that you want your smart machine to be able to identify, then click **Add as new tag**.
+   Tag names support alphanumeric characters, underscores, and hyphens.
+
+   For this tutorial, you will use the objects shown in the [Try Viam](https://app.viam.com/try) test octagon, which include shapes like a red star and an orange triangle.
+   The image below shows adding the `red_star` tag to an image:
+
+   {{< imgproc src="/tutorials/data-management/add-tag-red-star.png" alt="The tags and labels configuration pane of a selected image, with the text red_star entered as the tag" resize="350x" >}}
+
+1. Repeat this process for other images that contain objects you want your model to be able to identify.
+   Once you have added a tag to an image, you can select that tag from the **Tags** drop down menu for other images that also feature the tagged object.
+
+   {{< alert title="Tip" color="tip" >}}
+
+   For best results:
+
+   - Provide at least 10 images of the same object, taken from different angles, and repeat this approach for each object you want your smart machine to be able to identify.
+   - Include some images that do not contain any of the objects you wish to identify, but do not tag these images.
+   - If your smart machine operates in various lighting conditions, such as changing sunlight, include images of each object from varying lightning conditions.
+   - You can tag a single image with multiple tags if needed, but be sure to use `Multi label classification` when training your model later in this tutorial.
+
+   {{< /alert >}}
+
+Feel free to return to your smart machine's **Control** tab to position your camera to capture additional images from a variety of different angles, or with different lighting or background compositions.
+Generally, the more different perspectives of a given object you tag, the more likely your model will be able to identify it, even under differing conditions.
+The following is an example of a good selection of images containing the `blue_star` tag, taken from a variety of angles:
+
+{{< imgproc src="/tutorials/data-management/filter-by-blue-star.png" alt="The data tab showing images that contain blue stars from various angles, with a search filter for blue_star applied to only show images tagged with that label" resize="1200x" >}}
+
+If you want to remove a tag, click the **X** icon to the right of the tag name below the **Tags** field.
+
+For more information, see [Train a model](/manage/ml/train-model/).
+
+### Filter images using tags
+
+Once you have tagged your images, you can use those tags to filter your images to control which images to run your model on.
```

Comment:
```suggestion
Once you have tagged your images, you can use those tags to filter your images to control which images and run your model on.
```

tiny suggestion, it might change the intended meaning though which isn't my intention